src/java/htsjdk/tribble/BasicFeature.java

```diff
@@ -24,6 +24,8 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
+import htsjdk.samtools.util.SimpleInterval;
+
 /**
  * Trivial feature -- all it holds is the key information.
  *
```

Comment:
wouldn't it be nice if BasicFeature told you if it was 1-based closed ended, or is it supposed to be undefined?  If the former, document, if the latter, update the ```getInterval``` function.